tests/integration_tests/dashboards/api_tests.py

```diff
@@ -395,7 +395,7 @@ def test_get_dashboard_no_data_access(self):
         self.login(username="gamma")
         uri = f"api/v1/dashboard/{dashboard.id}"
         rv = self.client.get(uri)
-        self.assertEqual(rv.status_code, 200)
+        self.assertEqual(rv.status_code, 403)
```

Comment:
Nit (I know this was already using `self.assertEqual`, but might as well update it):

```suggestion
        assert rv.status_code == 403
```